packages/angular/generators.json

```
@@ -201,6 +207,12 @@
       "aliases": ["secondary-entry-point", "entry-point"],
       "description": "Creates a secondary entry point for an Angular publishable library."
     },
+    "mfe-remote": {
+      "factory": "./src/generators/mfe-remote/mfe-remote",
+      "schema": "./src/generators/mfe-remote/schema.json",
+      "aliases": ["remote"],
+      "description": "Generate a Remote Angular Micro-Frontend App."
```

nit: use full words for `app`, it reads better in the docs
```suggestion
      "description": "Generate a Remote Angular Micro-Frontend Application."
```